.github/workflows/publish.yml

```diff
@@ -19,11 +19,11 @@ jobs:
       - name: Grant execute permission for gradlew
         run: chmod +x gradlew
       - name: Publish to Repository
-        run: ./gradlew :ychat:clean publishReleasePublicationToYChatRepository
+        run: ./gradlew clean publish --no-parallel --stacktrace
 
         env:
-          MAVEN_USERNAME: ${{ secrets.MAVEN_USERNAME }}
```

Comment:
Hi @kikoso, I have updated the names of the environment variables to be more descriptive of what they represent. Could you update the corresponding GitHub environment keys with the following values?

MEMORY_KEY -> SIGNING_IN_MEMORY_KEY
SIGNINGKEY -> SIGNING_IN_MEMORY_KEY_ID
SIGNINGPASSWORD -> SIGNING_IN_MEMORY_KEY_PASSWORD